pkg/popeye.go

```diff
@@ -200,17 +201,26 @@ func (p *Popeye) Sanitize() error {
 				log.Fatal().Err(err).Msg("Closing report")
 			}
 		case isSetStr(p.flags.S3Bucket):
-			bucket, key, err := parseBucket(*p.flags.S3Bucket)
+			host, bucket, key, err := parseBucket(*p.flags.S3Bucket)
 			if err != nil {
 				log.Fatal().Err(err).Msg("Parse S3 bucket URI")
 			}
+			var s client2.ConfigProvider
 			// Create a single AWS session (we can re use this if we're uploading many files)
-			s, err := session.NewSession(&aws.Config{
-				LogLevel: aws.LogLevel(aws.LogDebugWithRequestErrors)})
-			if err != nil {
-				log.Fatal().Err(err).Msg("Create S3 Session")
-			}
-			// Create an uploader with the session and default options
+			if len(host) > 0 {
```

Comment:
I think this could be simplified the config is the thing that varies based on host info. So something like this could be better... 

```go
cfg := aws.Config{LogLevel: aws.LogLevel(aws.LogDebugWithRequestErrors)})}
if host != "" {
  cfg.Endpoint = aws.String(host)
}
s, err := session.NewSession(&cfg)
if err != nil {
  log.Fatal().Err(err).Msg("Create S3 Session")
}
```